libs/common/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 add_library(libs_common
     files.cpp
-    )
+    visitor.hpp)
```

Comment:
It is not necessary to include headers in targets.
http://voices.canonical.com/jussi.pakkanen/2013/03/26/a-list-of-common-cmake-antipatterns/
>Adding header files to target lists